src/App.tsx

```diff
@@ -1,57 +1,26 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Autocomplete } from './components/Autocompleted';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
 
   return (
     <main className="section">
       <h1 className="title">
-        {`${name} (${born} = ${died})`}
+        {selectedPerson
+          ? `${selectedPerson.name} ${selectedPerson.born} - ${selectedPerson.died}`
```

Comment:
Move this logic from jsx to the helper variable and use it here

Reply:
why?

Reply:
It's a bad practice write logic in the jsx, it's make your code more readable